.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Please remove all files `./idea/*` from commit

Reply:
We need __tests__/isBase64.test.js as well as we need other tests such as isUrl or isPath. I set up it but I used an image as a base64 data that is why it is 1Mb. I will try to use another small type of base64 data to solve that problem.